src/tribol/interface/tribol.cpp

```diff
@@ -457,28 +460,25 @@ void registerNodalVelocities( integer meshId,
                  "no mesh with id, " << meshId << "exists.");
 
    MeshData & mesh = meshManager.GetMeshInstance( meshId );
+   mesh.m_nodalFields.m_is_velocity_set = true;
+
    if (vx == nullptr || vy == nullptr)
    {
-      mesh.m_isValid = false;
+      mesh.m_nodalFields.m_is_velocity_set = false;
```

Comment:
should this be protected against numCells == 0?

Reply:
I don't think we need to guard it here. When we check for a valid coupling scheme, we don't perform all initialization checks if numCells == 0 so this boolean is never checked. If numCells == 0, we only look to see if this boolean is true for the gap-rate method that NEEDS it. Otherwise, we 'tolerate' or allow null velocity pointers.